src/reconcile/rec_write.c

```diff
@@ -2359,6 +2361,8 @@ __rec_build_delta(
         }
     } else if (F_ISSET(r->ref, WT_REF_FLAG_INTERNAL)) {
         /* The internal page delta would have already been built at this point if one exists. */
+        header = (WT_DELTA_HEADER *)r->delta.data;
```

Comment:
Should this be inside the size > 0 check?

Reply:
Thanks, updated in [b3b42c3](https://github.com/wiredtiger/wiredtiger/pull/11854/commits/b3b42c3d12ec3a5f68ffa94cd1ac4dab4119122a)